build/protobuf.mk

```diff
@@ -17,15 +17,14 @@
 # This file is evaluated in the repo's parent directory. See main.go's
 # go:generate invocation.
 
-ORG_ROOT       := .
-REPO_ROOT      := $(ORG_ROOT)/cockroach
-GITHUB_ROOT    := $(ORG_ROOT)/..
+REPO_ROOT      := ./cockroach
+GITHUB_ROOT    := $(REPO_ROOT)/vendor/github.com
```

Comment:
Ah, okay, I took it out since it seemed ambitious / potentially confusing whether it should refer to `GOPATH/src/github.com/cockroachdb` or to `./vendor/github.com/cockroachdb` -- I could see being surprised by it not being either of them.